docs/runtime/runtime-metadata.md

```diff
@@ -0,0 +1,196 @@
+---
+title: "The Metadata of Runtime"
+---
+
+In blockchain, the logic to process transactions and modify storage is usually called STF (State Transition Function). The more popular name among Substrate developers is runtime. 
```

Comment:
Remove

Reply:
I agree it should not belong here. But there is no similar statement in developer hub. My plan is to move it out to an `Introduction` page of runtime. But before that, we better keep it.

Reply:
We do explain in other places what a runtime and STF are. Even if we don't, so much of our documentation hinges on knowing what a blockchain is and our view of it (core components + STF) that everything would be meaningless without that knowledge. It really doesn't belong here.